lib/bitcoin/IProofOfFeeConfig.ts

```diff
@@ -0,0 +1,22 @@
+
+/**
+ * Defines configuration of approximate sliding windows used in
+ * proof of fee calculation.
+ */
+export interface ISlidingWindowQuantileConfig {
+  windowSizeInBatches: number;
+  batchSizeInBlocks: number;
+  feeApproximation: number;
```

Comment:
Can we add documentation on what `feeApproximation` mean? Or if it is hard to explain at this level without looking at ValueApproximator, is it really necessary to expose this as a configurable parameter? Is it possible to hardcode to a particular value?

Reply:
Yes, I have added. I think it is fine to hardcode this - but can we leave this on for now - we need to use this with many different values to get a good set of initial configuration values.